### PR TITLE
Scheduler 2.1

### PR DESCRIPTION
**Pull request by elijah-ward**

### CODE REVIEW

File: src/tso/scheduler/scheduler.py [hunk 1]
```diff
@@ -1,19 +1,87 @@
...
-    self.block_count += 1
 
+def create_transitioner(slew_rate):
+    return Transitioner(slew_rate,
+                        {'filter': {('B', 'G'): 10 * u.second,
+                                    ('G', 'R'): 10 * u.second,
+                                    'default': 30 * u.second}})
 
 
+def generate_schedule(schedule_horizon):
```
Comment (GAUNSD) [inline src/tso/scheduler/scheduler.py]: Just leaving this as a note if anyone jumps in to.
There will be more parameters to come soon.

Things like the observation Requests, and other user modifiable things

File: src/tso/scheduler/scheduler.py [hunk 2]
```diff
@@ -1,19 +1,87 @@
...
+
+    prior_scheduler = SequentialScheduler(constraints=global_constraints,
+                                          observer=cfht,
+                                          transitioner=transitioner)
+
+    priority_schedule = Schedule(noon_before, noon_after)
+
+    prior_scheduler(blocks, priority_schedule)
+
+    print(priority_schedule.to_table())
```
Comment (GAUNSD) [inline src/tso/scheduler/scheduler.py]: Could this be returned instead of being printed?

Comment (GAUNSD) [reply src/tso/scheduler/scheduler.py]: It can be printed or outputted to a file by an exporter of some kind

Comment (elijah-ward) [reply src/tso/scheduler/scheduler.py]: Absolutely. That will be a consideration once we figure out the flow for that stuff.

**Pull request merged**